src/Database/Connection.php

```diff
@@ -814,13 +814,35 @@ public function logQueries($enable = null)
     public function logger($instance = null)
     {
         if ($instance === null) {
-            if ($this->_logger === null) {
```

Comment:
Ahh, overlooked that 😇
@burzum minus one @ and an added blank line after `{@inheritDoc}`, yes 😄